source/common/src/tmsg.c

```diff
@@ -3159,7 +3162,7 @@ int32_t tSerializeSVDropTtlTableReq(void *buf, int32_t bufLen, SVDropTtlTableReq
   tEncoderInit(&encoder, buf, bufLen);
 
   if (tStartEncode(&encoder) < 0) return -1;
-  if (tEncodeI32(&encoder, pReq->timestamp) < 0) return -1;
+  if (tEncodeI64(&encoder, pReq->timestamp) < 0) return -1;
```

Comment:
Why change from i32 to i64? How can u ensure compatibility?

Reply:
To unify the unit to millisecond. Both ctime and btime use millisecond values.
But we can keep it in seconds for compatibility reasons.